go.mod

```diff
@@ -221,3 +221,5 @@ replace github.com/pingcap/tidb/parser => ./parser
 replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
 
 replace honnef.co/go/tools => honnef.co/go/tools v0.3.2
+
+replace github.com/tikv/client-go/v2 => github.com/joccau/client-go/v2 v2.0.0-rc.0.20220621023121-e36a43d8c763
```

Comment:
can we remove this replace?

Reply:
After https://github.com/tikv/client-go/pull/527 merged, I will update go.mode